src/axom/core/utilities/About.cpp.in

```diff
@@ -29,10 +31,7 @@ void about(std::ostream &oss)
   oss << "AXOM_VERSION_MAJOR: " << AXOM_VERSION_MAJOR << std::endl;
   oss << "AXOM_VERSION_MINOR: " << AXOM_VERSION_MINOR << std::endl;
   oss << "AXOM_VERSION_PATCH: " << AXOM_VERSION_PATCH << std::endl;
-
-#ifdef AXOM_VERSION_EXTRA
-  oss << "AXOM_VERSION_EXTRA: " << AXOM_VERSION_EXTRA << std::endl;
-#endif
+  oss << "AXOM_VERSION_EXTRA: " << gitSHA() << std::endl;
```

Comment:
Consistency: Should the string part of this line be `AXOM_GIT_SHA`  instead of the now removed `AXOM_VERSION_EXTRA`?

Reply:
I agree. I went back and forth but didn't want to change it for some odd reason I can't fully explain.